backend/userAPI/src/credential/credential.type.ts

```diff
@@ -0,0 +1,11 @@
+export const TsService =  {
+	GITHUB: "GITHUB",
+	GOOGLE: "GOOGLE",
+	SCALEWAY: "SCALEWAY",
+	COIN_MARKET: "COIN_MARKET",
+	DOCKER: "DOCKER",
+	ONE_DRIVE: "ONE_DRIVE",
```

Comment:
Please rename COIN_MARKET and ONE_DRIVE TO COINMARKET and ONEDRIVE respectively.

Reply:
Fixed in https://github.com/Cleopha/ifttt-like/pull/56/commits/d1ee44f61668dcfea60b8dad75745975d42889e3